utils/cobrautil/templates/help_flags_printer.go

```diff
@@ -95,13 +95,12 @@ func writeFlag(out io.Writer, f *flag.Flag, envPrefix string) {
 	fmt.Fprintf(out, getFlagFormat(f), f.Shorthand, f.Name, name, def, env, usage, deprecated)
 }
 
-var valueTypeRegex = regexp.MustCompile(`^[<|\[].+[>|\]]`)
-
 func flagNameAndUsage(f *flag.Flag) (string, string) {
```

Comment:
Why is it called NameAndUsage? This should be ValueAndUsage, shouldn't it?

Reply:
Library naming.